### DISCUSSION TITLE
Sql Server

### DISCUSSION THREAD

**Discussion by davepile** [Budibase/budibase#1360] [Help]
When adding queries to my MS Sql Server datasource I only have the read and create verbs available.

When I look at the Postgres datasource all four verbs are there.

The external data feature looks pretty new, so I am not sure if this is this a bug with the SQL Server connector or has it just not been implemented for Sql Server yet?

**Comment by joebudi** [comment]
Hi @davepile . Did you raise this issue in the Budibase Discord? If so, was it resolved?

**Reply by davepile** [reply]
No. I am not entirely clear on what the difference is between using the discussion here and the Discord channel.

If you think its better I raise issues there then I can do that (I have a postgres issue now that I have left sql server for the time being :) )

**Reply by joebudi** [reply]
Hey. Discussions are better for formal requests. Discord is better for ad-hoc chats.
